src/entities/protocols/seaport.ts

```diff
@@ -92,7 +107,7 @@ export class SeaportTrade extends NFTTrade<SeaportData> {
           100, // TODO: look into making this a better number
         ])
       }
-      planner.addCommand(CommandType.SEAPORT, [this.getTotalPrice().toString(), calldata], config.allowRevert)
+      planner.addCommand(this.commandType, [this.getTotalPrice().toString(), calldata], config.allowRevert)
```

Comment:
to add to my last comment: can this just be something like ` planner.addCommand(mapCmd(order.version))`